sql/src/test/java/org/apache/druid/sql/calcite/CalciteMultiValueStringQueryTest.java

```diff
@@ -1137,9 +1151,16 @@ public void testMultiValueListFilterComposed() throws Exception
                         .setContext(QUERY_CONTEXT_DEFAULT)
                         .build()
         ),
-        ImmutableList.of(
+        useDefault ? ImmutableList.of(
             new Object[]{0, 4L},
             new Object[]{1, 2L}
+        ) : ImmutableList.of(
+            // the fallback expression would actually produce 3 rows, 2 nulls, 2 0's, and 2 1s
+            // instead of 4 nulls and two 1's we get when using the 'native' list filtered virtual column
+            // this is because of slight differences between filter and the native
+            // selector, which treats a 0 length array as null instead of an empty array like is produced by filter
```

Comment:
This seems worth the release notes. Can you add some release note section in the PR description, so that we won't forget later?